src/main/java/com/poortorich/email/request/VerifyEmailCodeRequest.java

```diff
@@ -9,14 +10,14 @@
 @Getter
 @AllArgsConstructor
 public class VerifyEmailCodeRequest {
-    @NotBlank(message = "invalid_email")
-    @Email(message = "invalid_email")
+    @NotBlank(message = EmailResponseMessage.INVALID_EMAIL)
+    @Email
     private String email;
```

Comment:
```suggestion
    
    @NotBlank(message = EmailResponseMessage.INVALID_EMAIL)
    @Email
    private String email;
```

클래스 선언문이랑 변수 사이에 공백 넣어주세요 👀